### PR TITLE
Initial commit.

### PR DESCRIPTION
**Pull request by yakivmospan**
Changed launch icon for android.
Added splash screen for android.
Added basic theme.
Added basic home screen with basic navigation bar.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
-# coffee-shop
+# CoffeeShop
 Flutter based sample application.
+
```
Comment (ababicheva) [inline README.md:3]: Let's update README with some project details, e.g.:
- what is coffee shop app for?
- technologies that we are using
- code style/conventions

File: android/app/build.gradle [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        main.java.srcDirs += 'src/main/kotlin'
+    }
+
+    lintOptions {
+        disable 'InvalidPackage'
+    }
+
+    defaultConfig {
+        // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
+        applicationId "com.example.flutterapp"
```
Comment (ababicheva) [inline android/app/build.gradle]: let's use more company and project-specific app id

File: android/app/build.gradle [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        minSdkVersion 16
```
Comment (ababicheva) [inline android/app/build.gradle:42]: should we support from 16th api? let's finally start at least from 21st!

File: android/app/src/debug/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.flutterapp">
```
Comment (ababicheva) [inline android/app/src/debug/AndroidManifest.xml]: why a file from debug is in repo?

File: android/app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.flutterapp">
```
Comment (ababicheva) [inline android/app/src/main/AndroidManifest.xml]: package name - same as for app id

File: android/app/src/profile/AndroidManifest.xml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.example.flutterapp">
```
Comment (ababicheva) [inline android/app/src/profile/AndroidManifest.xml]: I didn't read getting started guides for flutter. what is profile folder for?

File: README.md (hunk 1 above)
Comment (yakivmospan) [reply README.md:3]: Yep, can be updated with some more stuff, thanks

File: android/app/build.gradle (hunk 2 above)
Comment (yakivmospan) [reply android/app/build.gradle]: 👍

File: android/app/build.gradle (hunk 3 above)
Comment (yakivmospan) [reply android/app/build.gradle:42]: This was autogenerated by flatter, I believe this is min that they support. I would like to keep it as it for the beginning and if needed change in future

https://github.com/flutter/flutter/issues/9310

File: android/app/src/debug/AndroidManifest.xml (hunk 4 above)
Comment (yakivmospan) [reply android/app/src/debug/AndroidManifest.xml]: This was autogenerated by flatter.

File: android/app/src/main/AndroidManifest.xml (hunk 5 above)
Comment (yakivmospan) [reply android/app/src/main/AndroidManifest.xml]: Yes, also need to update ios package as well

File: android/app/src/profile/AndroidManifest.xml (hunk 6 above)
Comment (yakivmospan) [reply android/app/src/profile/AndroidManifest.xml]: This was autogenerated by flatter.

>Use profile mode when you want to analyze performance.
https://flutter.dev/docs/testing/build-modes

File: android/app/src/debug/AndroidManifest.xml (hunk 4 above)
Comment (yakivmospan) [reply android/app/src/debug/AndroidManifest.xml]: Use debug mode during development, when you want to use hot reload.
https://flutter.dev/docs/testing/build-modes

File: android/app/build.gradle (hunk 3 above)
Comment (yakivmospan) [reply android/app/build.gradle:42]: https://flutter.dev/docs/resources/faq#what-devices-and-os-versions-does-flutter-run-on

File: ios/Flutter/flutter_export_environment.sh [hunk 7]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
```
Comment (oleksandrakovalova) [inline ios/Flutter/flutter_export_environment.sh]: Add this file to .gitignore

File: README.md (hunk 1 above)
Comment (ababicheva) [reply README.md:3]: please fix typos

File: ios/Flutter/flutter_export_environment.sh (hunk 7 above)
Comment (yakivmospan) [reply ios/Flutter/flutter_export_environment.sh]: @oleksandrakovalova updated

File: README.md (hunk 1 above)
Comment (yakivmospan) [reply README.md:3]: @budgieParrot updated

**Review by ababicheva — APPROVED** [review]

**Pull request merged**